### ISSUE TITLE
OS X Installation instructions unclear

### ISSUE THREAD

**Issue by coke** [wbraswell/rperl#90]
Trying to get a build of RPERL on OS X.

Downloaded https://github.com/wbraswell/rperl/tarball/master off the rperl.org site.

Reading INSTALL, I see:

```
For MacOSX & BSD Users:

    Download & run the RPerl Installer.
    Everywhere that calls the apt-get command, you will have to substitute your equivalent.
```

But when I download the installer, I don't understand the prompts; I guessed "29"; then "1" , and then I'm forced to choose an OS menu, at which point my options are Ubuntu or CentOS.

Having no OS X option, I pick 0 (Ubuntu). Tells me there's nothing to do on step 29, asks me to proceed to step 30. I pick yes, and the script exits, with no rperl in sight.

Any pointers on how to get a build going?

**Comment by wbraswell** [comment]
@coke 
Let's use this issue discussion thread to track your progress on installing RPerl inside OS X.
We got part of the way through the Perl installer, running sections 20 & 22.  We stopped on section 25 because it requires us to figure out which OS X packages are the equivalent of the existing Ubuntu packages:
https://github.com/wbraswell/rperl/blob/master/script/rperl_installer.sh#L612

Also, you were going to try to just run `cpanm RPerl` at this point and see what happens.  We had network problems with the Alien packages while we were on the wifi at the Salt Lake conference center, so please retry again and let me know what happens.

**Comment by coke** [comment]
Installing the modules went much better the next day. Ended up with these 4 deps failing to install:

==> Found dependencies: Math::GSL, Inline::C, Inline::Filters, Inline::CPP

Will continue to investigate those.

**Comment by coke** [comment]
So, reinstalling Inline::C ... mysteriously worked.

Down to Math::GSL, but stuck at:

https://github.com/leto/math--gsl/issues/151

**Comment by Util** [comment]
> On Jun 21, 2018, at 1:06 PM, Will Coleda <notifications@github.com> wrote:
> 
> Installing the modules went much better the next day. Ended up with these 4 deps failing to install:
> 
> ==> Found dependencies: Math::GSL, Inline::C, Inline::Filters, Inline::CPP
> 
> Will continue to investigate those.
> 
Math::GSL requires the GSL library, but does not use Alien::GSL, so you will need to `brew install gsl` or a MacPorts equiv, until Math::GSL starts using Alien, or maybe RPerl could require Alien::GSL before Math::GSL?

— 
Hope this helps,
Bruce Gray

**Comment by coke** [comment]
gsl is installed (via macports) I'm running the gsl-config in the linked ticket.

**Comment by wbraswell** [comment]
@Util 
Good point, somehow I missed the need for Math::GSL to depend upon Alien::GSL, I have made a new pull request to remedy this:
https://github.com/leto/math--gsl/pull/152

**Comment by wbraswell** [comment]
@coke 
Did Inline::Filters and Inline::CPP also install properly?

**Comment by coke** [comment]
Eventually. only the GSL dep is outstanding at this point.

**Comment by wbraswell** [comment]
@Util 
the Math::GSL pull request has been merged, so hopefully that will find its way onto CPAN at some point...
https://github.com/leto/math--gsl/pull/152

@coke 
Okay good at least we are down to just 1 installation issue now!  :-)
Please see my comment & request on the Math::GSL issue tracker:
https://github.com/leto/math--gsl/issues/151
